astroquery/gaia/__init__.py

```diff
@@ -39,6 +39,8 @@ class Conf(_config.ConfigNamespace):
                                       'MCMC_GSPPHOT',
                                       'MCMC_MSC']
 
+    GAIA_MESSAGES = _config.ConfigItem("notification?action=GetNotifications", "Gaia Messages")
```

Comment:
What are the other possibilities for this configitem?

Reply:
Sorry. Not sure if I have understood your question. This is the way we use to add a modifier to the request.

Reply:
If this configuration item can only take one value then it is not configurable and should not be managed by the configuration system. So if you have implemented this as a configuration item then there must be more values it can take, but they are not documented anywhere.

Reply:
If this configuration item can only take one value then it is not configurable and should not be managed by the configuration system. So if you have implemented this as a configuration item then there must be more values it can take, but they are not documented anywhere.

Reply:
This has to be deleted now